test.js

```diff
@@ -822,7 +823,7 @@ describe('standard-version', function () {
       shell.exec('git tag -a v1.0.0 -m "my awesome first release"')
       commit('feat: new feature!')
 
-      require('./index')({ silent: true })
+      require('./index')(Object.assign({}, defaults, { silent: true }))
```

Comment:
I believe manually importing and merging the defaults here is going to lead to false-positives in the tests.